### DISCUSSION TITLE
What about the term "security built in" - within Gatsby's project title in terms of using a CSP working only with 'unsafe-inline'?

### DISCUSSION THREAD

**Discussion by Mario8419** [gatsbyjs/gatsby#38624] [Community]
Hello,
wanted to ask very quickly whether there is something planned with fully CSP integration without using 'unsafe-inline' which pretty makes the CSP seemingly useless. Think there are a lot of people out there who wants to know whats the reason for gatsby users to can't have a fully secured gatsby site in terms of CSP. thx

**Comment by JustFly1984** [comment]
@Mario8419  sorry but why is that? What is the limitation?
